config/locales/en.yml

```diff
@@ -296,3 +297,10 @@ en:
         company: Company name
         department: Department name
         password: Password
+    profiles:
+      resource: Profiles
```

Comment:
Add translations for :ru.

Reply:
Are you sure? I thought we have no russian docs for API.